docs/spec.md

```diff
@@ -4018,6 +4096,7 @@ The remainder is calculated as `lhs - d * rhs`, where `d` is given by:
 * For floats: `division(lhs, rhs)` from IEEE-754 with rounding attribute
   `roundTowardZero`.
 * For complex numbers: TBD
+  ([#997](https://github.com/openxla/stablehlo/issues/997)).
```

Comment:
#997 not in frontend contract

Reply:
Same as [above](https://github.com/openxla/stablehlo/pull/1076#discussion_r1099053266).